Doc/library/argparse.rst

```diff
@@ -1808,7 +1808,9 @@ FileType objects
 Argument groups
 ^^^^^^^^^^^^^^^
 
-.. method:: ArgumentParser.add_argument_group(title=None, description=None)
+.. method:: ArgumentParser.add_argument_group(title, description, \
```

Comment:
@serhiy-storchaka Thanks for the feedback - I wasn't super familiar with the Sphinx syntax here so please do let me know if this is still incorrect.